app/models/payment/braintree_customer.rb

```diff
@@ -1,5 +1,9 @@
 class Payment::BraintreeCustomer < ActiveRecord::Base
   belongs_to :member
   has_many :braintree_payment_methods, class_name: Payment::BraintreePaymentMethod, primary_key: :customer_id, foreign_key: :customer_id
-  belongs_to :default_payment_method, class_name: Payment::BraintreePaymentMethod
+
+
+  def default_payment_method
+    braintree_payment_methods.order('created_at desc').first
+  end
```

Comment:
...and unless we want to change the logic around what payment method is default (we might eventually not want to use the latest method)